proxy/contracts/mainnet/MessageProxyForMainnet.sol

```diff
@@ -293,13 +307,14 @@ contract MessageProxyForMainnet is SkaleManagerClient, MessageProxy, IMessagePro
         require(hasRole(PAUSABLE_ROLE, msg.sender), "Incorrect sender");
         require(!pauseInfo[schainHash].paused, "Already paused");
         pauseInfo[schainHash].paused = true;
+        emit SchainPaused(schainHash);
```

Comment:
Also I would like to add unit tests to check that events are emitted

Reply:
Extended a test.